src/api_client.rs

```diff
@@ -121,14 +124,26 @@ pub enum RunStatus {
     Pending,
     Processing,
 }
+
 #[derive(Debug, Deserialize, Serialize)]
 #[serde(rename_all = "camelCase")]
-pub struct FetchLocalRunReportRun {
+pub struct FetchLocalRunReportHeadReport {
     pub id: String,
-    pub status: RunStatus,
-    pub url: String,
-    pub head_reports: Vec<FetchLocalRunReportHeadReport>,
+    pub impact: Option<f64>,
+    pub conclusion: ReportConclusion,
+}
+
+#[derive(Debug, Deserialize, Serialize)]
+pub struct FetchLocalRunReportResult {
```

Comment:
this struct is not related to the report but the result only. This be closer to our internal data structures:
```suggestion
pub struct FetchLocalRunBenchmarkResult {
```